install

```diff
@@ -30,7 +30,7 @@ fi
 mkdir "$BASEDIR/tanka" && cd "$BASEDIR/tanka"
 
 # Initialise Tanka
-tk init --k8s=1.22
+tk init --k8s=true
```

Comment:
It sounds like you can just omit the flag entirely:

```
tk init
```

Reply:
Apologies, I think we can ignore that... please refer to this comment:
https://github.com/grafana/tns/pull/45#discussion_r1002004508

Once I updated my Tanka, everything worked. But if you think it'd be better to omit the ```--k8s=1.22```, just let me know and I can try. But by default I'd rather leave things as they are because I don't understand them much 😄

Reply:
Nah, leaving things as-is sounds fine :)